.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
usually it is not .json file. please leave the name: '.eslintrc'. But anyway it works with error: node/no-unsupported-features/es-syntax
Try add:
"parserOptions": {
    "ecmaVersion": 2019,
    "sourceType": "module"
  }

Reply:
file extension removed
eslint is working now
thanks for this suggestion